data/brands/amenity/post_office.json

```diff
@@ -138,6 +138,141 @@
         "name": "The UPS Store",
         "short_name": "UPS"
       }
+    },
```

Comment:
I don't agree move China Post to a brand from operator because it is a kind of service